### PR TITLE
add radium v0.17.1 with npm auto-update

### PR DESCRIPTION
**Pull request by maruilian11**
**npm package url:** https://www.npmjs.com/package/radium
**63,083 downloads in the last month**
@PeterDaveHello  could u help me check this PR for #7128  ? thank you!
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update) 
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/radium/package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  "authors": [
+    "Alex Lande",
+    "Ian Obermiller"
+  ],
+  "npmName": "radium",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*.+(js|map)"
```
Comment (PeterDaveHello) [inline ajax/libs/radium/package.json]: Why not `"**/*"`?

Comment (maruilian11) [inline ajax/libs/radium/package.json]: there was `.gzip` in dist in an old case, i think both `**/*` and `**/*.+(js|map)` are fine.

Comment (PeterDaveHello) [inline ajax/libs/radium/package.json]: I didn't see the tgz file in the latest version.

Comment (maruilian11) [inline ajax/libs/radium/package.json]: it was another lib.

Comment (PeterDaveHello) [inline ajax/libs/radium/package.json]: So why don't you just fix it?

**Comment by maruilian11** [comment]
@PeterDaveHello done.

**Comment by PeterDaveHello** [comment]
This branch is 1 commit ahead, 417 commits behind cdnjs:master ...

**Comment by maruilian11** [comment]
@PeterDaveHello updated.

**Pull request merged**